### PR TITLE
Updated PyPairs to v3.0.9

### PR DESCRIPTION
**Pull request by rfechtner**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/pypairs/meta.yaml [hunk 1]
```diff
@@ -1,36 +1,60 @@
...
 build:
   number: 0
-  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
-  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  skip: True  # [py2k]
 
 requirements:
   host:
+    - anndata >=0.6.13
```
Comment (bgruening) [inline recipes/pypairs/meta.yaml]: I think most of these do not be in the host environment. I think only pip and python is needed.

File: recipes/pypairs/meta.yaml [hunk 2]
```diff
@@ -1,36 +1,60 @@
...
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 39546c372405f22cefd2b00fdbf0bb2bd6abe779d88a35f82f4b746807446bcb
+  sha256: baf268424ebe0eb58339e41877531c182e4c6634f59e9c21b77e0f33ec2f634d
 
 build:
   number: 0
-  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
-  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  skip: True  # [py2k]
```
Comment (bgruening) [inline recipes/pypairs/meta.yaml]: please leace noarch, but skip the python on the host/run level with `-python >3`

File: recipes/pypairs/meta.yaml [hunk 3]
```diff
@@ -1,36 +1,60 @@
...
+  license: BSD
+  license_family: BSD
+  license_file: LICENSE
+  summary: A python scRNA-Seq classifier
+  doc_url: https://pypairs.readthedocs.io/
+  dev_url: 
+
+extra:
+  recipe-maintainers:
+    - 11939699
```
Comment (bgruening) [inline recipes/pypairs/meta.yaml]: What is this? You can add your github user name if you like.

File: recipes/pypairs/meta.yaml [hunk 4]
```diff
@@ -1,36 +1,60 @@
...
   home: https://github.com/rfechtner/pypairs
-  license: MIT
-  license_family: MIT
-  summary: A Python-reimplementation of the Pairs algorithm described by A. Scialdone et al. (2015)
+  license: BSD
+  license_family: BSD
+  license_file: LICENSE
+  summary: A python scRNA-Seq classifier
+  doc_url: https://pypairs.readthedocs.io/
+  dev_url:
```
Comment (bgruening) [inline recipes/pypairs/meta.yaml]: please remove dev_url

File: recipes/pypairs/meta.yaml (hunk 2 above)
Comment (rfechtner) [reply recipes/pypairs/meta.yaml]: Ahh, got it. Sorry

**Comment by bgruening** [comment]
`ModuleNotFoundError: No module named 'matplotlib'` seems this is missing?

**Comment by rfechtner** [comment]
> `ModuleNotFoundError: No module named 'matplotlib'` seems this is missing?

Yes, indeed. Plotting is not fully completed, however this is atm not required. 
Will update this later

File: recipes/pypairs/meta.yaml (hunk 3 above)
Comment (rfechtner) [reply recipes/pypairs/meta.yaml]: Ok... I got this from running `conda skeleton pypi pypairs`. There the placeholder stated to fill in the user ID

File: recipes/pypairs/meta.yaml (hunk 4 above)
Comment (rfechtner) [reply recipes/pypairs/meta.yaml]: Sure

**Comment by bgruening** [comment]
Merci!

**Pull request merged**